test/libsolidity/smtCheckerTests/types/type_minmax.sol

```diff
@@ -0,0 +1,13 @@
+pragma experimental SMTChecker;
+
+contract C {
+	function f(uint a) public pure{
+		assert(a <= type(uint).max);
+		assert(a >= type(uint).min);
+		require(a <= type(uint64).max);
+		assert(a <= type(uint64).max);
+		assert(a <= type(uint32).max);
```

Comment:
This is the one which correctly triggers the assertion failure.

Reply:
Nice!

Reply:
Any other test ideas (besides the one going to the warning)?

Reply:
Not necessarily. What I started doing recently was copying tests from the semantic tests and replacing runs by assertions, if you think that makes sense here.

Reply:
Well there's a massive test case which has manual `require` statements for a large set of int types. Should that be copied, ugh?

Reply:
Haha no, I think this is fine

Reply:
I will add a handful examples with the actual value, just to be sure it has the correct numbers.

Reply:
Done. Copied from `semanticTests/integer`.